content/docs/0.6.0/reference/manifest_installation/index.md

```diff
@@ -68,29 +68,31 @@ kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/$KEPTNVERSION/ins
 
 ## Summary of applied images
 
+By following the *DockerHub* link, you will find the built image on DockerHub. The referenced Dockerfile shows how the image was built. 
+
 * Apply NATS resources:
-  * `connecteverything/nats-operator:0.6.0` - [DockerHub](https://hub.docker.com/layers/connecteverything/nats-operator/0.6.0/images/sha256-f83368baa5092a632c2e941ee7ba8cb6f925d0a068996a0a47ef4047edf2f12b?context=explore)
+  * `connecteverything/nats-operator:0.6.0` - [DockerHub](https://hub.docker.com/layers/connecteverything/nats-operator/0.6.0/images/sha256-f83368baa5092a632c2e941ee7ba8cb6f925d0a068996a0a47ef4047edf2f12b)
 
 * Install the Keptn Datastore:
   * `centos/mongodb-36-centos7:1` - [DockerHub](https://hub.docker.com/r/centos/mongodb-36-centos7)
-  * `keptn/mongodb-datastore:0.6.2` - [Dockerfile](https://github.com/keptn/keptn/blob/0.6.2/mongodb-datastore/Dockerfile)
-  * `keptn/distributor:0.6.2` - [Dockerfile](https://github.com/keptn/keptn/blob/0.6.2/distributor/Dockerfile)
+  * `keptn/mongodb-datastore:0.6.2` - [[DockerHub](https://hub.docker.com/repository/docker/keptn/mongodb-datastore/tags?page=1&name=0.6.2), [Dockerfile](https://github.com/keptn/keptn/blob/0.6.2/mongodb-datastore/Dockerfile)]
```

Comment:
I think the link to dockerhub you are using here requires authentication (at least it is asking me to login).

I tried and this link should work better: 
https://hub.docker.com/r/keptn/mongodb-datastore/tags?page=1&name=0.6.2

Alterantively, do we really need to point the user to the correct tag? I think it's not necessary, and it should be sufficient to point the user to the correct DockerHub repo: https://hub.docker.com/r/keptn/mongodb-datastore

Reply:
Thanks for the hint regarding the required authentication. Your suggested endpoint works fine in incognito mode. 

I would leave the reference (query parameter) for the tag because we already have the maintenance effort of updating the tag version in the github.com URL and point the user to the right tag reduces at least one obstacle when pulling the images manually.